src/plugins/intel_cpu/tests/functional/utils/quantization_utils.hpp

```diff
@@ -0,0 +1,58 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <unordered_map>
+#include "openvino/op/util/attr_types.hpp"
+
+namespace CPUTestUtils {
+
+struct QuantizationData {
+    float il;
+    float ih;
+    float ol;
+    float oh;
+    int levels = -1;  // -1 means that levels are not defined
+    bool isPerChannel = false;
+    ov::op::AutoBroadcastSpec auto_broadcast = ov::op::AutoBroadcastSpec(ov::op::AutoBroadcastType::NUMPY);
```

Comment:
I'd propose removing this param: in all real cases we have NUMPY broadcasting for FQ layers